File: app/src/main/java/nolambda/stiker/motionviews/widget/SimpleMotionViewCallback.java
package nolambda.stiker.motionviews.widget;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import nolambda.stiker.motionviews.widget.entity.MotionEntity;

public abstract class SimpleMotionViewCallback implements MotionViewCallback {
    @Override
    public void onEntityDoubleTap(@NonNull MotionEntity entity) {
    }

    @Override
    public void onEntitySelected(@Nullable MotionEntity entity) {
    }
}

File: app/src/main/java/nolambda/stiker/motionviews/widget/SimpleMotionViewCallback.kt
package nolambda.stiker.motionviews.widget

import nolambda.stiker.motionviews.widget.entity.MotionEntity

abstract class SimpleMotionViewCallback : MotionViewCallback {
    override fun onEntityDoubleTap(entity: MotionEntity) {}

    override fun onEntitySelected(entity: MotionEntity?) {}
}
